lib/chart/internal/utils.ex

```diff
@@ -1,24 +1,10 @@
 defmodule Chart.Internal.Utils do
   @moduledoc false
 
-  def add_prefix(key, nil) do
-    key
-  end
-
-  def add_prefix(key, prefix) when is_atom(prefix) and is_atom(key) do
-    String.to_atom("#{Atom.to_string(prefix)}_#{Atom.to_string(key)}")
-  end
-
-  def key_guard(kw, key, default_val, fun) do
-    fun.(Keyword.get(kw, key, default_val))
-  end
-
-  def key_guard(kw, key, default_val, boundary_tpl_vals, fun) do
-    fun.(Keyword.get(kw, key, default_val), boundary_tpl_vals)
-  end
-
-  def set_map(keywords, map) do
-    Enum.reduce(keywords, map, fn {key, val}, map -> Map.put(map, key, val) end)
+  def datetime_format(int, dt) when is_integer(int) and is_binary(dt) do
+    int
+    |> DateTime.from_unix!()
```

Comment:
Bude to vzdy `DateTime`? Nemuze to byt i `NaiveDateTime`?

Reply:
No mohl by, ale `NaiveDateTime` nema fci `from_unix()`. Resp jestli nekde existuje fce ktera by mi z unix time udelala `DateTime` nebo `NaiveDateTime` bylo by samozrejme lepsi ji zde pouzit.

Reply:
Spis je dulezite zjistit jaka data sem doputuji. `int` bude cislo > 0 a dt vypada jak a odkud jde?

Reply:
No nemyslim ze int bude cislo > 0. Nebot
```
DateTime.from_unix(0) 
{:ok, ~U[1970-01-01 00:00:00Z]}
```
A pro data starsiho to nejspis bude i zaporny. `dt`  bude string ruznyho [format](https://hexdocs.pm/nimble_strftime/NimbleStrftime.html)u.

Reply:
Trochu se bojim, ze uzivatel ti tam posle data v nejakem formatu a bude je v tom samem taky zobrazit.

Jinak pro `DateTime.from_unix()` plati nasledujici: 

> Unix times are always in UTC and therefore the DateTime will be returned in
UTC.

Reply:
jj tak by to melo fungovat. Tj. na zacatku si nastavi jaky chce format a ten dostane i na vystupu.